image/discourse_bench/Dockerfile

```diff
@@ -17,11 +19,12 @@ RUN gem install facter &&\
 
 # reconfigure PostgreSQL template databases to use UTF-8
 # otherwise rake db:create fails
-RUN pg_dropcluster --stop 9.3 main &&\
-    pg_createcluster --start -e UTF-8 9.3 main
+RUN pg_dropcluster --stop 9.5 main &&\
+    pg_createcluster --start -e UTF-8 9.5 main
 
 # create role "discourse"
 RUN /etc/init.d/postgresql start &&\
+    sleep 10 &&\
```

Comment:
Yeah... you're right.  I remembered seeing a "sleep 10" somewhere, but it was for redis.  Postgres has "sleep 5" in several places, so I switched to that instead.